src/requirements/EthereumAttestation/EthereumAttestationRequirement.tsx

```diff
@@ -0,0 +1,94 @@
+import { HStack, Img } from "@chakra-ui/react"
+import BlockExplorerUrl from "components/[guild]/Requirements/components/BlockExplorerUrl"
+import DataBlock from "components/[guild]/Requirements/components/DataBlock"
+import Requirement, {
+  RequirementProps,
+} from "components/[guild]/Requirements/components/Requirement"
+import { RequirementLinkButton } from "components/[guild]/Requirements/components/RequirementButton"
+import { useRequirementContext } from "components/[guild]/Requirements/components/RequirementContext"
+import { RPC } from "connectors"
+import useResolveAddress from "hooks/resolving/useResolveAddress"
+import shortenHex from "utils/shortenHex"
+
+export const EAS_SCAN_BASE = {
+  ARBITRUM: "https://arbitrum.easscan.org/schema/view",
+  OPTIMISM: "https://optimism.easscan.org/schema/view",
+  ETHEREUM: "https://easscan.org/schema/view",
+  SEPOLIA: "https://sepolia.easscan.org/schema/view",
+  BASE_GOERLI: "https://base-goerli.easscan.org/schema/view",
+} as const
+
+const EthereumAttestationRequirement = (props: RequirementProps): JSX.Element => {
+  const requirement = useRequirementContext()
+
+  const recipientDomain = useResolveAddress(requirement.data?.recipient)
+  const attesterDomain = useResolveAddress(requirement.data?.attester)
+
+  return (
+    <Requirement
+      image={
+        <Img
+          src={"/requirementLogos/eas.png"}
+          maxWidth={"var(--chakra-space-8)"}
+          maxHeight={"var(--chakra-space-8)"}
+        />
+      }
+      footer={
+        <HStack spacing={4}>
+          <RequirementLinkButton
+            href={`${EAS_SCAN_BASE[requirement?.chain ?? "ETHEREUM"]}/${
+              requirement?.data?.schemaId
+            }`}
+            imageUrl="/requirementLogos/eas.png"
+          >
+            {/* View schema on EAS scan */}
+            Schema
+          </RequirementLinkButton>
+          <BlockExplorerUrl
+            path="address"
+            address={requirement.data?.attester ?? requirement.data?.recipient}
+            // label={`View ${
+            //   requirement.type === "EAS_ATTEST" ? "recipient" : "attester"
+            // } on explorer`}
+            label={requirement.type === "EAS_ATTEST" ? "Recipient" : "Attester"}
+          />
+        </HStack>
+      }
+      {...props}
+    >
+      {requirement.type === "EAS_ATTEST" ? (
+        <>
+          Attest{" "}
+          <DataBlock>
```

Comment:
We could use the `DataBlockWithCopy` component for hex strings, so the users can easily copy it if needed, maybe that would improve the UX a bit.

Reply:
Nice! Thanks for mentioning this! At one point I was even thinking about implementing something like this, didn't notice, that it already has a nice component (then instead of implementing this, I just put the links in the footer 😄 ). Made the change in https://github.com/agoraxyz/guild.xyz/pull/873/commits/33278e1323f8aec7494cb2c008d0c969f0459f3b
Added this for the attester/recipient, and for the schema